### DISCUSSION TITLE
编译后固件出现问题进不去

### DISCUSSION THREAD

**Discussion by dd-bpir3** [immortalwrt/immortalwrt#1318] [Q&A]
![QQ图片20240531123235](https://github.com/immortalwrt/immortalwrt/assets/126221848/e249c677-aa97-4f4e-9840-51937ece0a4f)
编译后发现获取不到IP地址，手动配置地址进不去

**Comment by xlighting2017** [comment]
配置不对
具体你配置了什么，要问你自己
